content/sensu-go/5.18/release-notes.md

```diff
@@ -53,6 +54,28 @@ Read the [upgrade guide][1] for information about upgrading to the latest versio
 
 ---
 
+## 5.18.0 release notes
+
+**February 24, 2020** &mdash; The latest release of Sensu Go, version 5.18.0, is now available for download.
+**PLACEHOLDER FOR RELEASE SUMMARY**
+
+See the [upgrade guide][1] to upgrade Sensu to version 5.18.0.
+
+**IMPROVEMENTS:**
+
+- The `event.entity.entity_class` value now defaults to `proxy` for [`POST /events`][114] requests.
+- Proxy entities are now automatically created when events are published with an entity that does not exist.
+- Supported Bonsai asset versions are now prefixed with the letter `v` (for example, `v1.2.0`).
+- The version API now retrieves the Sensu agent version for the Sensu instance.
+- Sensu now reads and writes `initializationKey` to and from EtcdRoot, with legacy support (read-only) as a fallback.
+- Sensu will now check for an HTTP response other than `200 OK` response when fetching assets.
+- Updated Go version from 1.13.5 to 1.13.7.
+
+**FIXES:**
+
+- ([Commercial feature][115]) [Label selectors][116] and [field selectors][117] now accept single and double quotes to identify strings.
+- Fixed a bug that prevented wrapped resources from having their namespaces set by the default sensuctl configuration.
+
```

Comment:
```suggestion
- Improved event payload validation for the [events API][118] so that events that do not match the URL parameters on the `/events/:entity/:check` endpoint are rejected.
```

Reply:
💯